.circleci/config.yml

```diff
@@ -8,4 +8,4 @@ jobs:
       - checkout
       - run: pyenv local 2.7.13 3.5.2 3.6.0
       - run: make installdeps
-      - run: make lint && tox && codecov
+      - run: tox && make lint && codecov
```

Comment:
any reason for shifting order? I'd think we want things to fail faster

Reply:
My thought was that you'd want to know if your code passed the tests regardless of whether it passed linting. That is a more important indicator of how your branch is doing.

Reply:
In this case in particular, the lint error was masking the actual testing error that I cared about, and was not able to reproduce locally

Reply:
gotcha. this change makes sense to me